bundle/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml

```diff
@@ -64,10 +64,11 @@ metadata:
       ]
     capabilities: Basic Install
     categories: Database
-    description: The MongoDB Atlas Kubernetes Operator enables easy management of Clusters in MongoDB Atlas
-    operators.operatorframework.io/builder: operator-sdk-v1.4.2
-    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
-  name: mongodb-atlas-kubernetes.v0.7.0
+    description: The MongoDB Atlas Kubernetes Operator enables easy management of
+      Clusters in MongoDB Atlas
+    operators.operatorframework.io/builder: operator-sdk-v1.16.0
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
+  name: mongodb-atlas-kubernetes.v0.5.0
```

Comment:
This is suspicious, the version shouldn't change ☝️

Reply:
Fixed. Let's add this folder to `.gitignore` in a separate PR

Reply:
Sure